Makefile

```diff
@@ -430,8 +432,42 @@ e2e-deploy-gatekeeper: e2e-helm-install
     --set mutatingWebhookTimeoutSeconds=2 \
     --set auditInterval=0
 
+e2e-deploy-base-ratify: e2e-notaryv2-setup e2e-notation-leaf-cert-setup e2e-inlinecert-setup
+	docker build --progress=plain --no-cache \
+	-f ./httpserver/Dockerfile \
+	-t localbuild:test .
+	kind load docker-image --name kind localbuild:test
+
+	docker build --progress=plain --no-cache --build-arg KUBE_VERSION=${KUBERNETES_VERSION} --build-arg TARGETOS="linux" --build-arg TARGETARCH="amd64" -f crd.Dockerfile -t localbuildcrd:test ./charts/ratify/crds
```

Comment:
wondering if CRD build can be shared between the two

Reply:
that's good point. I tried to do that, but found out `localbuild` and `localbuildcrd` share the same tag while deploying. If this block is moved out and shared between 2 targets, we would need to pass in a new argument to targets, which would involve more files.

Reply:
synced offline, updated image name so that we can move this block out as a separate target.